src/Docker/Manager/ContainerManager.php

```diff
@@ -172,6 +172,32 @@ public function start(Container $container, array $hostConfig = [])
         return $this;
     }
 
+
+    /**
+     * Copy files or folders from a container
+     *
+     * @param \Docker\Container $container
+     * @param                   $resource file or folder name
+     *
+     * @throws \Docker\Exception\UnexpectedStatusCodeException
+     *
+     * @return tarfile stream
```

Comment:
`@return \Guzzle\Stream\Stream Tarfile stream` would be more accurate and allows for autocompletion.